### PR TITLE
Code smell

### PR DESCRIPTION
**Pull request by daveaugustus1**
Signed-off-by: Pappu Kumar <pappu.kumar@progress.com>
Fixes are based on SonarCloud code smell notifications.
 
### :nut_and_bolt: Description: What code changed, and why?
- Spell check
- Constants definition for repeated strings
- Unzip function restructure 
 

rebuild components/infra-proxy-service

### :white_check_mark: Checklist

**All PRs** must tick these:

- [X] I have read the [CONTRIBUTING document](https://github.com/chef/automate/blob/master/CONTRIBUTING.md).
- [X] All commits signed-off for the [Developer Certification of Origin](https://github.com/chef/chef/blob/master/CONTRIBUTING.md#developer-certification-of-origin-dco).

With occasional exceptions, all PRs **from Progress employees** must tick these:

- [X] Is the code clear? *(complicated code or lots of comments--subdivide and use well-named methods, meaningful variable names, etc.)*
- [X] Consistency checked? *(user notifications, user prompts, visual patterns, code patterns, variable names)*
- [X] Repeated code blocks eliminated? *(adapt and reuse existing components, blocks, functions, etc.)*
- [X] Spelling, grammar, typos checked? *(at a minimum use `make spell` in any component directory)*
- [X] Code well-formatted? *(indents, line breaks, etc. improve rather than hinder readability)*

All PRs **from Progress employees** should tick these if appropriate:

- [X] Tests added/updated? (all new code needs new tests)
- [X] Docs added/updated? (all customer-facing changes)

*Please add a note next to any checkbox above if you are NOT ticking it.*

### CODE REVIEW

**Review by kalroy — APPROVED** [review]

File: components/infra-proxy-service/server/orgs_test.go [hunk 1]
```diff
@@ -1043,6 +1043,50 @@ func TestOrgs(t *testing.T) {
...
+		test.ResetState(context.Background(), t, serviceRef)
+		serverRes, err := cl.CreateServer(ctx, &request.CreateServer{
+			Id:        constants.TestServerId,
+			Name:      "Chef Infra Server",
+			Fqdn:      "domain.com",
+			IpAddress: "",
+		})
+		require.NoError(t, err)
+		require.NotNil(t, serverRes)
+		// TODO: Need to add the webui key with server after the webui key changes in create server API
```
Comment (YashviJain01) [inline components/infra-proxy-service/server/orgs_test.go:1057]: Can you please remove the "TODO" string, it creates a code smell itself

File: components/infra-proxy-service/migrations/migrations.go [hunk 2]
```diff
@@ -331,3 +313,12 @@ func (s *MigrationServer) ConfirmPreview(ctx context.Context, req *request.Confi
 		MigrationId: req.MigrationId,
 	}, nil
 }
+
+func StreamErr(err error, ctx context.Context, stream service.MigrationDataService_UploadFileServer, migServer *MigrationServer, migrationId, serverId, errMsg string) {
+	log.Errorf(errMsg)
```
Comment (YashviJain01) [inline components/infra-proxy-service/migrations/migrations.go:318]: Can you print the error message properly, like "Unable to upload the zip file %s',err.Errors()) and not take the string in the function itself

File: components/infra-proxy-service/migrations/migrations.go [hunk 3]
```diff
@@ -331,3 +313,12 @@ func (s *MigrationServer) ConfirmPreview(ctx context.Context, req *request.Confi
...
+	res := handleErrorForUploadFileAndMigration(err, migrationId, serverId, migServer, ctx)
+	errStream := stream.SendAndClose(res)
+	if errStream != nil {
+		log.Errorf(errMsg)
```
Comment (YashviJain01) [inline components/infra-proxy-service/migrations/migrations.go:322]: here you have log the errStream.Error(), with the lo unable to close the stream with this error.

Comment (dave-augustus) [reply components/infra-proxy-service/migrations/migrations.go:322]: I need some more info on this.

**Pull request closed without merging**